src/js/SubmissionContainer.jsx

```diff
@@ -52,22 +53,26 @@ var SubmissionContainer = React.createClass({
       )
     }
 
-    if(code > 3) progress = <p>Progress component goes here</p>
+    if(code > 3) progress = <p className="third">Progress component goes here</p>
 
-    if(code > 6) editsContainer = <EditsContainer/>
+    if(code > 5) editsContainer = <EditsContainer/>
```

Comment:
At one point we had discussed not running quality and macro until syntactical and validity were complete and correct. If that were possible then we should be ok?

@jmarin - thoughts on adding those extra states to be able to show some edits as those edit type checks complete?